### ISSUE TITLE
Restore old search behaviour or make it optionally available

### ISSUE THREAD

**Issue by polyamAdmin** [polyamspace/mastodon#220]
### Pitch

Either discard upstream's new search of all indexable public toots, or add an admin option to restore old search behaviour of only returning toots which were interacted with.

### Motivation

I don't really see any value in the new search and it only appeases to large instances.

**Comment by polyamAdmin** [comment]
Apparently vanilla is already doing it in 26760 by adding a search param, which will only search own and interacted with toots.
Gawd, I wish they would add better descriptions...

**Comment by polyamAdmin** [comment]
The mentioned PR has been merged and is included in #223.
I might still want to reverse it, so it searches the private index by default, or implement an option, but I'm not able to do that now.

**Comment by polyamAdmin** [comment]
The following patch should search in the private index by default unless "all" is specified.
I can't test this currently.
 
```diff
index a45ae3d09b..31d95c239f 100644
--- a/app/lib/search_query_transformer.rb
+++ b/app/lib/search_query_transformer.rb
@@ -56,10 +56,10 @@ class SearchQueryTransformer < Parslet::Transform

     def indexes
       case @flags['in']
-      when 'library'
-        [StatusesIndex]
-      else
+      when 'all'
         [PublicStatusesIndex, StatusesIndex]
+      else
+        [StatusesIndex]
       end
     end

```
